docs/en/writing-running-appium/caps.md

```diff
@@ -106,6 +106,7 @@ These Capabilities are available only on Android-based drivers (like
 |`chromedriverPorts`| A list of valid ports for Appium to use for communication with Chromedrivers. This capability supports multiple webview scenarios. The form of this capability is an array of numeric ports, where array items can themselves be arrays of length 2, where the first element is the start of an inclusive range and the second is the end. By default, Appium will use any free port.|e.g. `[8000, [9000, 9005]]`|
 |`ensureWebviewsHavePages`| Whether or not Appium should augment its webview detection with page detection, guaranteeing that any webview contexts which show up in the context list have active pages. This can prevent an error if a context is selected where Chromedriver cannot find any pages. Defaults to `false`|e.g. `true`|
 |`webviewDevtoolsPort`| To support the `ensureWebviewsHavePages` feature, it is necessary to open a TCP port for communication with the webview on the device under test. This capability allows overriding of the default port of `9222`, in case multiple sessions are running simultaneously (to avoid port clash), or in case the default port is not appropriate for your system.|e.g. `9543`|
+| `enableWebviewDetailsCollection` | Prints WebView detailed information via Chrome Debugger Protocol endpoint as the Appium log since Appium 1.18.0+. It helps to confirm the version of WebView the device under test has, for example. Defaults to `false` | `true` or `false` |
```

Comment:
Prints -> Enables collection of detailed WebView information via /json/version CDP endpoint
This helps to properly match Chromedriver version which supports the given WebView. Without this feature enabled Appium tries to guess the version of the WebView based on the corresponding version of the corresponding installed package having (which usually [fails](https://github.com/appium/appium/issues/13918) for custom web views).